transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemRootResourceDefinition.java

```diff
@@ -248,6 +254,8 @@
 
     private final boolean registerRuntimeOnly;
 
+    public static int maximum_timeout = MAXIMUM_TIMEOUT.getDefaultValue().asInt();
```

Comment:
As well as my other comment, the naming of this field is not consistent with our code style